__tests__/__unit__/ZoweNode.unit.test.ts

```diff
@@ -90,6 +90,7 @@ describe("Unit Tests (Jest)", () => {
             new ZoweDatasetNode("BRTVS99.CA11.SPFTEMP0.CNTL", vscode.TreeItemCollapsibleState.Collapsed,
                                 rootNode, null, undefined, undefined, profileOne),
             new ZoweDatasetNode("BRTVS99.DDIR", vscode.TreeItemCollapsibleState.Collapsed, rootNode, null, undefined, undefined, profileOne),
+            new ZoweDatasetNode("BRTVS99.VS1", vscode.TreeItemCollapsibleState.None, rootNode, null, VSAM_CONTEXT, undefined, profileOne)
```

Comment:
I'm always going to find it a bit strange to have to specify multiple `null` or `undefined` just to provide a 5th or 6th parameter. 😋 

No changes requested though!